source/comply/legal-hold.rst

```diff
@@ -1,152 +1,174 @@
 
-3 - Leverage compliance API in your DLP tool of choice
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Access a Legal Hold secret
+:::::::::::::::::::::::::::
 
-Connect it directly to the Amazon S3 bucket where compliance exports are stored. Our recommended DLP tool of choice is `Smarsh <https://www.smarsh.com/>`_. Learn more about their legal hold & e-discovery processes in their `documentation <https://www.smarsh.com/platform/enterprise/discovery>`_.
+A Legal Hold secret enables you to verify the authenticity of the data for a Legal Hold in Mattermost.
 
-You may optionally also use other tools that connect with Amazon S3, such as Active360, AwareHQ, Onna, or Trellix, though our team will be able to provide limited support using these tools.
+Select the **Show** |preview-icon| icon to display the Legal Hold secret key. Keep a copy of this key in a secure location.
 
-4 - (Optional) Configure a data retention policy
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. image:: ../images/legal-hold-secret.png
+  :alt: An example of a Legal Hold Secret Key available to Mattermost system admins.
 
-This is typically implemented alongside legal hold, though is not required.
+To verfiy the contents of the files in this Legal Hold, you must append the processor command with the following flag: ``--legal-hold-secret <KEY>``. The output verifies the file and returns the authenticity state of files along with the rest of the output for the processor, as follows:
 
-1. In Mattermost, go to **System Console > Compliance > Data Retention Policies**.
-2. Set either a **Global retention policy** that applies to all messages and files older in your system, or a **custom retention policy** for specific teams or channels.
-3. Learn more about data retention configuration options in our :doc:`product documentation </comply/data-retention-policy>`.
+Success:
 
-Step 4: Identify custodians
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
 
-Select the individuals or user groups that you want to place on legal hold.
+  Secret key was provided, verifying legal holds...
+  - Verifying Legal Hold *processor9*: Verified
 
-Step 5: Initiate legal hold
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Error:
+
+.. code-block:: text
+
+  ...
+  Secret key was provided, verifying legal holds...
+  - Verifying Legal Hold *processor9*: [Error] hash mismatch for file: legal_hold/processor9_i7k1dbkipiyojeess6ozi4agyr/index.json
+  ...
 
-Use the built-in functionality in Smarsh to specify one or more custodians for legal hold, and the number of days the custodian(s) are placed on legal hold.
+Download Legal Hold data
+:::::::::::::::::::::::::
+
+Select the **Download** |download-icon| icon to download a copy of the preserved data to a location on your local machine. Note, no data will be available to download until at least one scheduled job is completed. 
```

Comment:
```suggestion
Select the **Download** |download-icon| icon to download a copy of the preserved data to a location on your local machine. Note, no data will be available to download until at least one scheduled job is completed. This may take up to 24 hours.
```